chart/templates/rbac.yaml

```diff
@@ -145,3 +146,17 @@ rules:
       - get
       - update
       - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Why do we need this ?

Reply:
don't we need the binding?

This is coming from `chart/templates/clusterrolebinding.yaml` which got deleted and merged here.